specification/quota/resource-manager/Microsoft.Quota/preview/2023-06-01-preview/groupquota.json

```diff
@@ -0,0 +1,1797 @@
+          },
+          {
+            "$ref": "#/parameters/ScopeInParameters"
+          },
+          {
+            "$ref": "#/parameters/GroupQuotaNameInParameters"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success, with the details of the groupQuota request.",
+            "schema": {
+              "$ref": "#/definitions/SubmittedResourceRequestStatusList"
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Management/managementGroups/{mgId}/{resourceProviderScope}/providers/Microsoft.Quota/groupQuotas/{groupQuotaName}/groupQuotaRequests/{requestId}": {
+      "get": {
+        "tags": [
+          "GroupQuotaLimits-Requests"
+        ],
+        "summary": "Get the status of a single GroupQuota request by requestId.",
+        "description": "Get API to check the status of a GroupQuota request by requestId.  Use the polling API - OperationsStatus URI specified in Azure-AsyncOperation header field, with retry-after duration in seconds to check the intermediate status. This API provides the finals status with the request details and status.",
+        "operationId": "GroupQuotaLimits-Requests_Get",
+        "x-ms-examples": {
+          "GroupQuotaLimits-Requests_Get": {
+            "$ref": "./examples/GroupQuotaLimits-Requests/GroupQuotaLimits-Requests_Get.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/MgIdInParameters"
+          },
+          {
+            "$ref": "#/parameters/ScopeInParameters"
+          },
+          {
+            "$ref": "#/parameters/GroupQuotaNameInParameters"
+          },
+          {
+            "$ref": "#/parameters/RequestIdInParameters"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success, with the details of the groupQuota request.",
+            "schema": {
+              "$ref": "#/definitions/SubmittedResourceRequestStatus"
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Management/managementGroups/{mgId}/subscriptions/{subscriptionId}/{resourceProviderScope}/providers/Microsoft.Quota/groupQuotas/{groupQuotaName}/quotaAllocations": {
```

Comment:
>/providers/Microsoft.Management/managementGroups/{mgId}/subscriptions/{subscriptionId}/{resourceProviderScope}/providers/Microsoft.Quota/groupQuotas/{groupQuotaName}/quotaAllocations [](http://example.com/codeflow?start=5&length=182)

In one of the above APIs, it was /providers/Microsoft.Quota/subscriptions/{subscriptionId}, and here it is /subscriptions/{subscriptionId}/{resourceProviderScope}/providers/Microsoft.Quota/, can you please explain the relationship between a subscription and groupQuota?

Reply:
In the previous case subscription was added to the groupQuota, so it's an input parameter.

In the quota allocation API - The quota is allocated to the subscription, that's why, it's part of the resource URI. Setting the resource context - 
In this case - We want to list all the QuotaAllocations to the Subscription - SubA from GroupQuota {groupQuotaName} for the specific RP/Location (based on the RP's resource Path, as explained earlier).